packages/dotcom-build-code-splitting/src/bundleTypes.ts

```diff
@@ -16,6 +16,8 @@ interface IBundleWithRegExp {
   usedInUnknownWay?: boolean
 }
 
+const isJS = (m) => m.type && m.type.startsWith('javascript/')
```

Comment:
We should stick to convention of calling a thing by its full name, i.e. `m` -> `module`.

Reply:
https://github.com/Financial-Times/dotcom-page-kit/blob/master/contribution.md#user-content-general-style

> Avoid abbreviating names (e.g. `request` is better than `req`)